docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -1,132 +1,84 @@
-    kind: VirtualService
-    metadata:
-      name: httpbin
-      namespace: $NAMESPACE
-    spec:
-      hosts:
-      - "httpbin.$DOMAIN_TO_EXPOSE_WORKLOADS"
-      gateways:
-      - $GATEWAY
-      http:
-      - match:
-        - uri:
-            prefix: /
-        route:
-        - destination:
-            port:
-              number: 8000
-            host: httpbin.$NAMESPACE.svc.cluster.local
-    EOF
+    kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"numTrustedProxies": NUM_OF_TRUSTED_PROXIES}}}'
     ```
+    Due to the variety of network topologies, the Istio CR must specify the configuration property **numTrustedProxies**, so that the client IP address can be extracted correctly.
 
-### Verify the X-Forwarded-For and X-Envoy-External-Address Headers
-1. Get your public IP address.
-    ```bash
-    curl -s https://api.ipify.org
-    ```
+2. If you use a GCP or Azure cluster, run the following command to set the traffic policy to Local. If you use a different cloud service provider, skip this step.
 
-2. Send a request to the HttpBin workload.
     ```bash
-    curl -s "https://httpbin.$DOMAIN_TO_EXPOSE_WORKLOADS/get?show_env=true"
+    kubectl patch istios/default -n kyma-system --type merge -p '{"spec":{"config":{"gatewayExternalTrafficPolicy": "Local"}}}'
     ```
-3. Verify that the response contains the **X-Forwarded-For** and **X-Envoy-External-Address** headers with your public IP address, for example:
-    ```json
-    {
-      "args": {
-        "show_env": "true"
-      },
-      "headers": {
-        "Accept": "...",
-        "Host": "...",
-        "User-Agent": "...",
-        "X-Envoy-Attempt-Count": "...",
-        "X-Envoy-External-Address": "165.1.187.197",
-        "X-Forwarded-Client-Cert": "...",
-        "X-Forwarded-For": "165.1.187.197",
-        "X-Forwarded-Proto": "...",
-        "X-Request-Id": "..."
-      },
-      "origin": "165.1.187.197",
-      "url": "..."
-    }
-    ``` 
+    >[!WARNING]
+    > For production Deployments, it is strongly recommended to deploy Istio Ingress Gateway Pod to multiple nodes if you `enable externalTrafficPolicy : Local`. For more information, see [Network Load Balancer](https://istio.io/latest/docs/tasks/security/authorization/authz-ingress/#network).
+    >
+    >Default Istio installation profile configures **PodAntiAffinity** to ensure that Ingress Gateway Pods are evenly spread across all nodes. This guarantees that the above requirement is satisfied if your IngressGateway autoscaling configuration **minReplicas** is equal to or greater than the number of nodes. You can configure autoscaling options in the Istio CR using the field **spec.config.components.ingressGateway.k8s.hpaSpec.minReplicas**.
+    >[!TIP]
+    > If you use a GCP or Azure cluster, you can find your load balancer's IP address in the field **status.loadBalancer.ingress of the ingress-gateway** Service.
+<!-- tabs:end -->
```

Comment:
The Warning and Tim are not displayed correctly. Isn't it because Markdown and HTML do not work well together? Maybe adding an empty line between 57 and 58 could help?

Reply:
Adding an empty line helped, thank you